### DISCUSSION TITLE
ch_PP-OCRv4_rec文本识别，模型训练前40epoch时，验证精度可以达到90%以上，现在验证精度接近0.

### DISCUSSION THREAD

**Discussion by banhongjun** [PaddlePaddle/PaddleOCR#12265] [Q&A]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/99791142/083cdd1e-ff38-4ec9-947b-f0eea64b2d93)
请问这种问题咋回事？

**Comment by gengyanlei** [comment]
@banhongjun 
你好，请问你能直接训练ch_PP-OCRv4么，我按照官网上的config、预训练模型参数 执行出错。请问你是如何微调v4的det的呢

**Comment by banhongjun** [comment]
这是识别模型，检测问题有人提出过，自己搜索下。

**Comment by gengyanlei** [comment]
@banhongjun 嗯嗯，检测的配置参数修改设置这些搜到了。就是训练精度  loss都为0的问题
